airbyte-cdk/python/setup.py

```diff
@@ -56,7 +56,29 @@
         "Deprecated~=1.2",
     ],
     python_requires=">=3.7.0",
-    extras_require={"dev": ["MyPy~=0.812", "pytest", "pytest-cov", "pytest-mock", "requests-mock"]},
+    extras_require={
+        "dev": ["MyPy~=0.812", "pytest", "pytest-cov", "pytest-mock", "requests-mock"],
+        "sphinx-docs": [
+            "Jinja2~=3.0",
+            "MarkupSafe~=2.0",
+            "Pygments~=2.10",
+            "Babel~=2.9",
+            "docutils~=0.17",
+            "imagesize~=1.2",
+            "packaging~=21.0",
+            "pyparsing~=2.4",
+            "pytz~=2021.1",
+            "snowballstemmer~=2.1",
+            "Sphinx~=4.2",
+            "sphinxcontrib-applehelp~=1.0",
+            "sphinxcontrib-devhelp~=1.0",
+            "sphinxcontrib-htmlhelp~=2.0",
+            "sphinxcontrib-jsmath~=1.0",
+            "sphinxcontrib-qthelp~=1.0",
+            "sphinx-rtd-theme~=1.0",
+            "sphinxcontrib-serializinghtml~=1.1",
```

Comment:
I believe you just put dependencies from Sphinx 4.2 here, if so, please remove it, you only need Sphinx 4.2 and all extra plugins